modules/websocket_bridge/src/websocket_bridge/utils/ws_protocol.cpp

```diff
@@ -27,10 +27,6 @@ auto convertIpcRawServiceResponseToWsServiceResponse(
     WsServiceId service_id, WsServiceCallId call_id,
     const ipc::zenoh::ServiceResponse<std::vector<std::byte>>& raw_response, WsServiceResponse& ws_response)
     -> bool {
-  if (raw_response.value.empty()) {
-    return false;
-  }
-
```

Comment:
note that when a service call returns an empty proto object or a proto object has all optional fields unset, I assumed that even then the byte array would never be empty, but turns out it is. So this is a valid response.